vmaas/webapp/app.py

```diff
@@ -434,7 +412,17 @@ class RPMPkgNamesHandlerGet(BaseHandler):
     @classmethod
     async def get(cls, rpm=None, **kwargs):
         """RPM list or Content Set list by SRPM list or RPM list"""
-        return await cls.handle_request(cls.rpm_pkg_names_api, 1, 'rpm_name_list', rpm, **kwargs)
+        return await cls.handle_request(cls.rpm_pkg_names_api, 1, None, 'rpm_name_list', rpm, **kwargs)
+
+
+def metrics():
+    """Provide current prometheus metrics"""
+    return generate_latest(), 200, {'Content-Type': 'text/plain; charset=utf-8'}
+
+
+def dummy_200():
+    """Dummy endpoint returning 200"""
+    return "", 200
```

Comment:
looks like a forgotten testing function

Reply:
No, it's used for OPTIONS /v3/cves endpoint. Not sure if it's still used by something, I've just converted it.